tests/mod.rs

```diff
@@ -586,3 +586,34 @@ fn custom_trait_bound() {
 
 	Something::<NotEncode, u32>::decode(&mut &encoded[..]).unwrap();
 }
+
+#[test]
```

Comment:
I htink this needs a `#[cfg(feature = "bit-vec")]`, otehrwise `cargo test` fails locally (or it needs to move to the bit_vec file) :)